Update README.md (#1621)
spiffe/spire: README.md

- [Get SPIRE](#get-spire)
- [Learn about SPIRE](#learn-about-spire)
- [Integrate with SPIRE](#integrate-about-spire)
- [Contribute to SPIRE](contribute-to-spire)
- [Further Reading](further-reading)
- [Security](security)

SPIRE is hosted by the [Cloud Native Computing Foundation](https://cncf.io) (CNCF) as a sandbox level project. If you are an organization that wants to help shape the evolution of technologies that are container-packaged, dynamically-scheduled and microservices-oriented, consider joining the CNCF. For details read the CNCF [announcement](https://www.cncf.io/blog/2018/03/29/cncf-to-host-the-spiffe-project/).

## Get SPIRE

> - Pre-built releases of SPIRE can be found at [https://github.com/spiffe/spire/releases](https://github.com/spiffe/spire/releases). These releases contain both SPIRE Server and SPIRE Agent binaries plus the officially supported plugins.

Reviewer: We can get rid of some of the following language since plugins are not built into the binaries and aren't shipped separately.
 ```suggestion
- Pre-built releases of SPIRE can be found at [https://github.com/spiffe/spire/releases](https://github.com/spiffe/spire/releases). These releases contain both SPIRE Server and SPIRE Agent binaries.
```

- Alternatively, you can [build SPIRE from source](/CONTRIBUTING.md).

## Learn about SPIRE

- Before trying SPIRE, it is recommended to become familiar with its [architecture](https://spiffe.io/spire/) and design goals.
- Once ready to get started, see [Getting Started Guides](https://spiffe.io/spire/try/).

> - The [SPIRE Server](https://github.com/spiffe/spire/blob/master/doc/spire_server.md) and [SPIRE Agent](https://github.com/spiffe/spire/blob/master/doc/spire_agent.md) reference guides covers in more detail the specific configuration options and plugins available.

Reviewer: nit: the server and agent doc links can be relative

- There are several examples demonstrating SPIRE usage in the [spire-examples](https://github.com/spiffe/spire-examples) repository.
- Check [ADOPTERS.md](./ADOPTERS.md) for a list of production SPIRE adopters, a view of the ecosystem, and use cases.
- See the [SPIRE Roadmap](https://github.com/spiffe/spire/wiki/Roadmap) for a list of planned features and enhancements.
- Join Slack. If you have any questions about how SPIRE works, or how to get it up and running, the best place to ask questions is the [SPIFFE Slack Organization](https://slack.spiffe.io/).

## Integrate with SPIRE

> - See [Extend SPIRE](https://spiffe.io/spire/docs/extending/) to learn about the SPIRE highly extensible plugin framework.

Reviewer: ```suggestion
- See [Extend SPIRE](https://spiffe.io/spire/docs/extending/) to learn about the highly extensible SPIRE plugin framework.
```

## Contribute to SPIRE

The SPIFFE community maintains the SPIRE project. Information on the various SIGs and relevant standards can be found in
https://github.com/spiffe/spiffe.

- See [CONTRIBUTING](https://github.com/spiffe/spire/blob/master/CONTRIBUTING.md) to get started.
- Use [GitHub Issues](https://github.com/spiffe/spire/issues) to request features or file bugs.
- See [GOVERNANCE](https://github.com/spiffe/spiffe/blob/master/GOVERNANCE.md) for SPIFFE and SPIRE governance policies.

## Further Reading

- The [Scaling SPIRE guide](/doc/scaling_spire.md) covers design guidelines, recommendations, and deployment models.
- For an explanation of how SPIRE compares to related systems such as secret stores, identity providers, authorization policy engines and service meshes see [comparisons](https://spiffe.io/spire/comparisons/).

## Security

### Security Assessments

> The [CNCF Special Interest Group for Security]([https://github.com/cncf/sig-security](https://github.com/cncf/sig-security)) has conducted two assessments on SPIFFE and SPIRE design and configuration with respect to security.  The following links contain summary reports as well as the threat modeling material produced as part of the asssessment processes:

Reviewer: ```suggestion
The [CNCF Special Interest Group for Security]([https://github.com/cncf/sig-security](https://github.com/cncf/sig-security)) has conducted two assessments on SPIFFE and SPIRE design and configuration with respect to security.  The following links contain summary reports as well as the threat modeling material produced as part of the assessment:
```

- [Scrutinizing SPIRE to Sensibly Strengthen SPIFFE Security](https://blog.scytale.io/scrutinizing-spire-security-9c82ba542019)
- [SIG Security SPIFFE/SPIRE Security Assessment Summary](https://github.com/cncf/sig-security/tree/master/assessments/projects/spiffe-spire)
- [SIG Security SPIFFE/SPIRE Security Assessment Document](https://github.com/cncf/sig-security/blob/master/assessments/projects/spiffe-spire/self-assessment.md)

### Reporting Security Vulnerabilities

> If you've found a vulnerability or a potential vulnerability in SPIRE please let us know at security@spiffe.io. We'll send a confirmation email to acknowledge your report, and we'll send an additional email when we've identified the issue positively or negatively.

Reviewer 2: Maybe this can be reworded and expanded in a way that we express how important and instrumental is security in the SPIRE project, and we encourage to report security issues.
Aligned with that, it would be good if we (at some point) provide a PGP public key for the communication with us.

Author: Agreed. 

Ideally this can submitted on the web using HTTPS (TLS) or an email encrypted using OpenPGP.

In order to maintain the projects CII best practices badge, it must be ensured that project's initial response time for any vulnerability report received is less than or equal to 14 days.
